en/docs/install-and-setup/setup/distributed-deployment/product-profiles.md

```diff
@@ -220,6 +223,8 @@ Create an optimized distribution for a particular API-M profile.
     
     ```  
         
+If you have encountered any issue relevant to loading the axis2 configuration, check if all the files are available in the `<PRODUCT-HOME>/repository/resources/conf/templates/repository/conf/axis2` directory and it tallies with the backup directory. If not, please add the missing J2 files accordingly.
```

Comment:
Shall we mention the <PRODUCT-HOME>/ repository/resources/conf/templates/repository/conf/registry_TM.xml.j2 file also ?

This is outside the axis2 directory.

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/6215/commits/0c0f0becf96542fbc506712404aef53efa128892